products/arangodb

```diff
@@ -0,0 +1,33 @@
+---
+title: ArangoDB
+category: db
+permalink: /arangodb
+releasePolicyLink: https://arangodb.com/master-services-agreement-inc-may-2023-cs/#eolpolicy
+changelogTemplate: "https://docs.arangodb.com/stable/release-notes/version-__RELEASE_CYCLE__/"
+releaseLabel: "ArangoDB __RELEASE_CYCLE__"
+eolWarnThreshold: 121
```

Comment:
```suggestion
eolWarnThreshold: 121

auto:
-   git: https://github.com/arangodb/arangodb.git
```

With this config latest releases and release dates would be automatically updated by our automation, and maintainers would be warned of new releases. Automation in this case would work with tags: https://github.com/arangodb/arangodb/tags.

Reply:
Will add it.